src/algebra/group_power/basic.lean

```diff
@@ -231,9 +225,20 @@ variables [comm_monoid M] [add_comm_monoid A]
 theorem mul_pow (a b : M) (n : ℕ) : (a * b)^n = a^n * b^n :=
 (commute.all a b).mul_pow n
 
-@[to_additive nsmul.is_add_monoid_hom]
-instance pow.is_monoid_hom (n : ℕ) : is_monoid_hom ((^ n) : M → M) :=
-{ map_mul := λ _ _, mul_pow _ _ _, map_one := one_pow _ }
+-- at the time of writing, the call to `simps` below generates `pow_add_monoid_hom_apply`
+-- instead of `nsmul_add_monoid_hom_apply`
+
+/-- The `n`th power map on a commutative monoid for a natural `n`, considered as a morphism of
+monoids. -/
+@[to_additive nsmul_add_monoid_hom "Multiplication by a natural `n` on a commutative additive
+monoid, considered as a morphism of additive monoids."]
```

Comment:
```suggestion
@[to_additive nsmul_add_monoid_hom "Multiplication by a natural `n` on a commutative additive
monoid, considered as a morphism of additive monoids.", simps]
```
This generates simp-lemmas for both the multiplicative and additive version, but it has the wrong name for the additive version (the statement should be correct). 
The name is fixed by #8457.
My suggestion is to just generate the simp lemmas (with the wrong name), and then the name will automatically be corrected once #8457 lands.